tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -1,304 +1,392 @@
+  <data name="DevDriveOptimized" xml:space="preserve">
+    <value>Optimized</value>
+    <comment>Dev drive optimized indicator</comment>
+  </data>
+  <data name="DevDriveSizeText" xml:space="preserve">
+    <value>Size - {0} {1}</value>
+    <comment>Dev drive size</comment>
+  </data>
+  <data name="DevDriveUsedSizeText" xml:space="preserve">
+    <value>{0} {1} used</value>
+    <comment>Dev drive size used</comment>
+  </data>
+  <data name="DevDriveVolumesHeader.Text" xml:space="preserve">
+    <value>Dev Drive volumes</value>
+    <comment>Header for Dev Drive volumes list in Dev Drive insights page</comment>
+  </data>
+  <data name="DevDriveTrusted.Text" xml:space="preserve">
+    <value>Trusted</value>
+    <comment>Dev drive is trusted</comment>
+  </data>
+  <data name="DevDriveUntrusted.Text" xml:space="preserve">
+    <value>Untrusted</value>
+    <comment>Dev drive is not trusted</comment>
+  </data>
+  <data name="FileExplorerCard.Description" xml:space="preserve">
+    <value>Adjust these settings for a more developer-friendly experience using File Explorer.</value>
+    <comment>The description for the File Explorer settings card</comment>
+  </data>
+  <data name="FileExplorerCard.Header" xml:space="preserve">
+    <value>File Explorer</value>
+    <comment>The header for the File Explorer settings card</comment>
+  </data>
+  <data name="FileExplorer_Header" xml:space="preserve">
+    <value>File Explorer</value>
+    <comment>Header for the File Explorer page in the breadcrumb bar</comment>
+  </data>
+  <data name="EndTaskOnTaskBar.Description" xml:space="preserve">
+    <value>Enable end task in taskbar by right click</value>
+    <comment>The description for the end task on task bar settings card</comment>
+  </data>
+  <data name="ExampleText" xml:space="preserve">
+    <value>Example: </value>
+    <comment>Example string, will be followed by a sample location to move the cache to a dev drive location</comment>
+  </data>
+  <data name="EndTaskOnTaskBar.Header" xml:space="preserve">
+    <value>End Task</value>
+    <comment>The header for the end task on task bar settings card</comment>
+  </data>
+  <data name="MainPage_Header" xml:space="preserve">
+    <value>Windows customization</value>
+    <comment>Header for main Windows customization page in the breadcrumb bar</comment>
+  </data>
+  <data name="MakeChangesText" xml:space="preserve">
+    <value>Make changes</value>
+    <comment>Make changes button on dialog</comment>
+  </data>
+  <data name="MakeTheChangeText" xml:space="preserve">
+    <value>Make the change</value>
+    <comment>Make the change button on optimizer card</comment>
+  </data>
+  <data name="MakeTrusted.Text" xml:space="preserve">
+    <value>Trust</value>
+    <comment>Make dev drive trusted</comment>
+  </data>
+  <data name="MoreWindowsSettingsSectionDescription.Text" xml:space="preserve">
+    <value>Additional settings below can be configured in the system settings app</value>

```

Comment:
This description becomes invalid when the UI changes.  Can the direction be removed?